packages/victory-core/src/index.d.ts

```diff
@@ -462,7 +462,7 @@ export type DomainPaddingPropType =
       y?: PaddingType;
     };
 
-export type RangeTuple = [number, number];
+export type RangeTuple = [number, number] | [Date, Date];
```

Comment:
Wasn't sure about this, but it seems like Date would be a commonly used value here as well? Looks like it is used in demo

Reply:
The naming on this is a bit odd... If it were only used for range rather than also being used for domain it could be strictly [number, number], but, you're right that domains could include dates too.

Reply:
I ended up replacing the use of `RangeTuple` in `VictoryBrushContainerProps` and `VictoryZoomContainerProps` to use `DomainTuple`. https://github.com/FormidableLabs/victory/blob/master/packages/victory-core/src/index.d.ts#L451